README.md

```diff
@@ -148,6 +148,8 @@ These variables are used by the service startup scripts in the Docker images, bu
 * `POLLER_THREADS`: The number of threads to run in the poller process. The default is 5.
 * `CONNECT_TIMEOUT`: The timeout for establishing a network connection to the BigBlueButton server in the load balancer and poller in seconds. Default is 5 seconds. Floating point numbers can be used for timeouts less than 1 second.
 * `RESPONSE_TIMEOUT`: The timeout to wait for a response after sending a request to the BigBlueButton server in the load balancer and poller in seconds. Default is 10 seconds. Floating point numbers can be used for timeouts less than 1 second.
+* `LOAD_MIN_USER_COUNT`: Minimum user count of a meeting, used for calculating server load. Defaults to 15.
+* `LOAD_JOIN_BUFFER_TIME`: The time(in minutes) until the `LOAD_MIN_USER_COUNT` will be used for calculating server load. Defaults to 15.
```

Comment:
Hmm. The wording here is confusing or backwards - it implies that LOAD_MIN_USER_COUNT is not applied for the first 15 minutes, but then is applied afterwards.

I'm not sure the best way to word these help messages would be, maybe it would help to reverse the order, something like this?
```suggestion
* `LOAD_JOIN_BUFFER_TIME`: During the buffer time after a meeting starts, the server load calculation accounts a boosted load value to the meeting to compensate for people who have not yet joined. Values is in minutes, defaults to 15.
* `LOAD_MIN_USER_COUNT`: The minimum number of people to assume a meeting will have during the join buffer time. Defaults to 15.
```

Reply:
hmm, may be @jfederico can help

Reply:
@kepstin I would agree with you. I didn't get it with the readme explanation and had the need to look into the code.